app/components/instant-article-field/instant-article-config.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+angular.module('cms.instant_article.config', []).factory('InstantArticleConfig', [function () {
+  var supportedFeatureTypes = [];
+
+  var factory = {};
+
+  factory.setSupportedFeatureTypes = function(featureTypes) {
+    supportedFeatureTypes = featureTypes;
+
+    return this;
+  };
+
+  factory.getSupportedFeatureTypes = function() {
+    return supportedFeatureTypes;
+  };
+
+  return factory;
+}]);
```

Comment:
I'll buy it for consistency but it's a common pattern to also set variables at run time.